tools/ui-components/src/alert/alert.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import { Story } from '@storybook/react';
+import React from 'react';
+import { Alert, AlertProps } from './alert';
+
+const story = {
+  title: 'Example/Alert',
+  component: Alert
+};
+
+const Template: Story<AlertProps> = args => <Alert {...args} />;
```

Comment:
Do you mean something like this:
```tsx
export const TrueStory = (): JSX.Element => {
  const [show, setShow] = React.useState(false);

  return (
    <div>
      <button onClick={() => setShow(true)}>Show Alert</button>
      {show && (
        <Alert onDismiss={() => setShow(false)} variant='danger'>
          Something went wrong
        </Alert>
      )}
    </div>
  );
};
```

So that we can actually see the alert visibility being toggled?

Reply:
To my understanding, the alert component should internally have the close functionality so it would work once it is imported in another component. 

In the Bootstrap alert component, there is an in internal transition wrapper which should handle the display. It would be great if you could research that and replicate a similar functionality so Alert would work out of the box.

Reply:
> To my understanding, the alert component should internally have the close functionality so it would work once it is imported in another component.

Yeah, I see your point. I haven't implemented that in our alert, because it seemed like we're not using anywhere that functionality. When looking for "Alert" references in our codebase, I found that most of the time the alert component is used without closing functionality, and the only time we actually pass `onDismiss` handler to it, is in Flash component, which handles both closing logic and transitions:

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/src/components/Flash/index.tsx

![flash and alert](https://user-images.githubusercontent.com/58401630/140798214-334bc4b3-4362-40ff-a29d-26f485a0655a.png)

What do you think?

Reply:
I decided to keep them separate for now. I think that we should try to keep the transition to Tailwind as seamless as possible, ideally it should be possible to just swap the imports, and it seems like combining Flash and Alert wouldn't be so straightforward.